_src/app/NewCollectionEvent.js

```diff
@@ -190,12 +190,18 @@ define([
             this.clearReport();
 
             domClass.remove(this.successMsgContainer, 'hidden');
-            window.scrollTo(0, 0);
+
+            window.setTimeout(() => {
+                window.scrollTo({
+                    top: 0,
+                    behavior: 'smooth'
+                });
+            }, 500);
 
             var that = this;
             window.setTimeout(function hideSuccessMsg() {
```

Comment:
maybe a close button is better suited for this?

Reply:
You're probably right: 4222073